### DISCUSSION TITLE
Multiple R sessions

### DISCUSSION THREAD

**Discussion by pawelru** [posit-dev/positron#3877] [Q&A] [answered]
Is it possible to work on multiple R sessions at the same time?

I'm working on multiple packages / projects at the same time. My typical VSCode setup usually looks like this:
![image](https://github.com/posit-dev/positron/assets/12943682/8a05485d-2297-4798-891c-be92cc621751)

How to achieve this in Positron? Is this possible?

(Please feel free to redirect me to existing discussion / issue if it's there. I did my best looking for similar topic but I haven't found any)

**Comment by jthomasmock** [comment, marked as answer]
Thanks for interest! I would add on to: https://github.com/posit-dev/positron/issues/3173 which covers an ask for multiple sessions, both interactive or in the background

**Reply by pawelru** [reply]
thanks for the hint!
upvoted and subscribed for news on this!

**Reply by jthomasmock** [reply]
Of course! Do you mind selecting this as the answer if it covers your question?

**Reply by pawelru** [reply]
Hmm strange. I can't see "Mark as answer" button. Feel free to do this yourself if you can
![image](https://github.com/posit-dev/positron/assets/12943682/0cc06f31-a27f-449d-8d7e-fb3f2cb6d49d)
